render/topologies.go

```diff
@@ -171,6 +171,25 @@ var ContainerImageRenderer = Map{
 	},
 }
 
+// ContainerHostnameRenderer is a Renderer which produces a renderable container
+// by hostname graph..
+var ContainerHostnameRenderer = Map{
+	MapFunc: MapCountContainers,
+	Renderer: Map{
+		MapFunc: MapContainer2Hostname,
+		Renderer: MakeReduce(
+			Map{
+				MapFunc:  MapContainer2ContainerImage,
+				Renderer: ContainerRenderer,
+			},
+			Map{
+				MapFunc:  MapContainerImageIdentity,
+				Renderer: SelectContainerImage,
+			},
+		),
+	},
+}
```

Comment:
Are we just grouping containers by hostname, or by container image, then hostname? It looks like the latter, but checking whether that is what we want.